lib/client.js

```diff
@@ -161,9 +162,11 @@ var GiantSwarm = (function () {
       if (clusterId) {
         headers[CLUSTER_ID_HEADER] = clusterId;
       }
-      request
-        .get(apiEndpoint + uri)
-        .set(headers)
+      var r = request.get(apiEndpoint + uri);
+      if (params !== null) {
```

Comment:
Or default the params further above to an empty string. The `ifs` feel unnecessary